bower.json

```diff
@@ -39,9 +42,17 @@
     "angular-sanitize": "1.3.0 - 1.5.*",
     "angular-bootstrap": "0.13.x",
     "lodash": "3.x",
-    "patternfly": "~3.11.0"
+    "patternfly": "a014002ebcfad2c6ff2fbeda52dfcd425e016f2a"
```

Comment:
We can't use #master, but I changed this to master-dist for clarity instead of using the commit.